versions.lock

```diff
@@ -209,8 +209,11 @@ org.apache.commons:commons-lang3:3.9 (11 constraints: 48a8c3cb)
 org.apache.commons:commons-math3:3.2 (2 constraints: 90196ff8)
 org.apache.commons:commons-pool2:2.4.2 (1 constraints: 0a050336)
 org.apache.commons:commons-text:1.8 (1 constraints: b4102e9b)
-org.apache.httpcomponents:httpclient:4.5.12 (2 constraints: 3a20244b)
-org.apache.httpcomponents:httpcore:4.4.13 (3 constraints: 913094d1)
+org.apache.httpcomponents:httpclient:4.2.5 (1 constraints: 3c0ca8f8)
+org.apache.httpcomponents:httpcore:4.2.4 (2 constraints: 611ce6cf)
+org.apache.httpcomponents.client5:httpclient5:5.0.1 (1 constraints: ca13906e)
+org.apache.httpcomponents.core5:httpcore5:5.0.1 (3 constraints: 57394c0b)
+org.apache.httpcomponents.core5:httpcore5-h2:5.0.1 (1 constraints: 3a13043c)
```

Comment:
Note people are already using this in production, because they've picked up latest dialogue faster than your repo did.